packages/endpoints-organizations-registrar/src/entities/organizations/adapters/send-email-notifications.js

```diff
@@ -162,14 +163,31 @@ const initSendEmailNotifications = (initCtx) => {
     const inviterOrganization = invitation?.inviterId
       ? await context.repos.organizations.findById(invitation.inviterId)
       : null;
-    await initCtx.sendEmail(
-      emailToSignatoryForOrganizationApproval({
+    const emailTemplateSignatoryApproval = compile(
```

Comment:
This line of code is very low-level, and its location is wrong as the compile is only needed once. It's worth adding a utility function to handle loading these handlebars templates. They are "side-effects" in the same way that loading from the db, or calling a http endpoint, and should be handled outside of the main business logic

Reply:
kicked it out to `@fastify/view`, see [82ab97a](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/82ab97ae2694ce71f4ae1d2ca0962ad5df4f63af)